app/src/main/res/layout/fragment_timetable_item_details_page.xml

```diff
@@ -8,30 +8,22 @@
 	android:clickable="true"
 	android:focusable="true"
 	android:orientation="vertical"
-	android:paddingBottom="12dp">
+	android:padding="12dp">
```

Comment:
This causes the press-animation for absences and lesson topic items to not extend to the border of the screen.
Is there another way to solve this?

Reply:
Oh, you're right. I didn't realize that.